src/goto-symex/show_program.cpp

```diff
@@ -17,12 +17,13 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include <langapi/language_util.h>
 
-
 void show_program(const namespacet &ns, const symex_target_equationt &equation)
 {
-  unsigned count=1;
+  unsigned count = 1;
 
-  std::cout << "\n" << "Program constraints:" << "\n";
+  std::cout << "\n"
```

Comment:
If you're doing some cleanup work here, I'd suggest to replace all occurrences of `"\n"` by `'\n'` (single character instead of string).